coral/docs/mock-api-for-development.md

```diff
@@ -0,0 +1,53 @@
+# Mocking an API for development
```

Comment:
Amazing resource, thank you!

I feel there is a little bit of confusion though, because we have

> Also, please take care not to commit these mocks 🙏.

And then a few lines later

> If there is no mock already, you can create a new one. The mock will be useful for testing later!

If I understand correctly, what should not be committed is the **usage of the mocks to replace actual API calls**, not the mocks themselves. Sorry if it sounds like pedantic semantics, but it tripped me up ^^

Maybe the title could be " Using API endpoints mocks for local development"? And rewording so that it's clear that people should be careful not to commit such "development only" code?

Reply:
Yep, that's a good idea! Now that I read it again I realize that's a bit confusing 😬 
I will take another go at it on Monday ☕

Reply:
@mathieu-anderson Updated! Maybe you can take a look again.